  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint0_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_chassis_joint
    pid: {p: 2.0, i: 0.0, d: 0.0}

  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_1_joint
    pid: {p: 2.0, i: 0.0, d: 0.0}

  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm_2_joint
    pid: {p: 2.0, i: 0.0, d: 0.0}

  jointC_position_controller:
    type: effort_controllers/JointPositionController
    joint: RGB_joint
    pid: {p: 2.0, i: 0.0, d: 0.0}
